packages/fiori-freestyle/templates/worklist/add/webapp/controller/v4.0/App.controller.js

```diff
@@ -0,0 +1,14 @@
+sap.ui.define([
```

Comment:
Can we use packages/fiori-freestyle/templates/worklist/add/webapp/controller/v4/App.controller.js
instead of packages/fiori-freestyle/templates/worklist/add/webapp/controller/v4.0/App.controller.js ?

Reply:
The reason its this way is to support a direct mapping to odata versions. We will have 4.1 soon.